### ISSUE TITLE
Peripheral myelinating corneal nerve fibers

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10877]
Preferred term label:
Peripheral myelinating corneal nerve fibers

Synonyms
Peripheral corneal nerve myelination, Myelinated corneal nerves

Definition (free text, please give PubMed ID)
The presence of myelinated nerve fibers in the peripheral cornea, an uncommon finding typically observed in association with systemic or ocular conditions, or as an isolated benign anomaly.

Parent term (use hpo.jax.org/app)
Abnormality of the corneal stroma (HP:11492)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
None identified

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Is this a synonym of [Prominent corneal nerve fibers HP:0010726](https://hpo.jax.org/browse/term/HP:0010726)? If not could we specify what the difference is?

**Comment by daboe01** [comment]
ups, sorry for the dupe.

**Comment by pnrobinson** [comment]
@daboe01 thanks! I added the synonyms.
